### PR TITLE
Fix update message when msgid changed

### PR DESCRIPTION
**Pull request by zukkun**
I found problem what update message when `msgid` changed.
I fixed it therefore. Please confirm.
Thank you.

### CODE REVIEW

File: i18n-msg.html [hunk 1]
```diff
@@ -96,7 +96,8 @@
        */
       msgid: {
         type: String,
-        value: null
+        value: null,
+        observer: '_messageChanged'
```
Comment (ebidel) [inline i18n-msg.html]: let's use "_msgidChanged"

**Comment by ebidel** [comment]
Reviewed

**Comment by zukkun** [comment]
@ebidel I changed observer name. Thank you.

**Comment by ebidel** [comment]
LGTM

**Pull request merged**